util/extract.php

```diff
@@ -9,8 +9,8 @@
 	foreach($files as $file) {
 		$str = file_get_contents($file);
 
-		$pat = '| t\(([^\)]*)\)|';
-		$patt = '| tt\(([^\)]*)\)|';
+		$pat = '| L10n::t\(([^\)]*)\)|';
+		$patt = '| L10n::tt\(([^\)]*)\)|';
```

Comment:
The generated files need a `use Friendica\Core\L10n;` call now 😅

Reply:
Hmmm, I am unclear on this. Am I adding that in to the pat(t) strings here?

Reply:
The `$pat(t)` strings are just regular expressions, you should add it lower, after the PHP opening tag, in the `$s` variable.

Reply:
I'm sorry, I'm dumb, there's no use for a `use` since we don't call the methods, we're just looking for them on the PHP files.